backend/src/main/java/gov/cdc/usds/simplereport/api/organization/OrganizationMutationResolver.java

```diff
@@ -426,9 +428,19 @@ public OrganizationQueueItem markPendingOrganizationAsDeleted(
   }
 
   /** Support-only mutation to mark an organization as deleted. This is a soft deletion only. */
+  @Transactional
   @MutationMapping
   public Organization markOrganizationAsDeleted(
       @Argument UUID organizationId, @Argument boolean deleted) {
+    Organization orgToBeDeleted = organizationService.getOrganizationById(organizationId);
+    List<ApiUser> usersInOrgToBeDeleted =
+        apiUserService.getAllActiveUsersByOrganization(orgToBeDeleted);
```

Comment:
This fails when `deleted` param is `false` because the `getAllActiveUsersByOrganization` method throws the following error "Cannot get Okta users from nonexistent organization."

Any thoughts on what to do here for the reverse situation? Or does markOrganization as deleted need to only delete orgs and a separate method needs to undelete them?